modules/renderer/photos.js

```diff
@@ -7,7 +7,7 @@ import { utilRebind } from '../util/rebind';
 
 export function rendererPhotos(context) {
     var dispatch = d3_dispatch('change');
-    var _layerIDs = ['streetside', 'mapillary', 'mapillary-map-features', 'mapillary-signs', 'openstreetcam'];
+    var _layerIDs = ['streetside', 'mapillary', 'mapillary-map-features', 'mapillary-signs', 'openstreetcam', 'rapid-features'];
```

Comment:
`rapid-features` is a little generic for the layer name. how about `rapid-mapillary-features`?

Reply:
Mahmoud, let me take a close look at this to make sure the name is consistent everywhere else in the code.